cmd/kubeadm/app/cmd/phases/init/uploadconfig.go

```diff
@@ -96,6 +96,7 @@ func NewUploadConfigPhase() workflow.Phase {
 func getUploadConfigPhaseFlags() []string {
 	return []string{
 		options.CfgPath,
+		options.NodeCRISocket,
```

Comment:
`NodeCRISocket` is required to patch node `kubeadm.alpha.kubernetes.io/cri-socket` annotation.